ts/WoltLabSuite/Core/Component/Ckeditor.ts

```diff
@@ -309,6 +309,14 @@ export async function setupCkeditor(
     configuration.licenseKey = licenseKey;
   }
 
+  const { DATABASE_FOR_AUTO_COMPLETE } = await import("./EmojiPicker/woltlab-core-emoji-picker");
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-expect-error
+  // TODO remove eslint-disable
```

Comment:
`// TODO`

Also, this module should import `woltlab-core-emoji-picker` as a side-effect and thus preemptively loading the emoji data.